tests/FSharp.Compiler.ComponentTests/CompilerService/AsyncMemoize.fs

```diff
@@ -126,8 +128,6 @@ let ``We can disconnect a request from a running job`` () =
             Started, key
             Finished, key ]
 
-[<Fact>]
-let ``We can cancel a job`` () =
```

Comment:
Just curious, what was driving the test split? Not that this should necessarily stay as it was.

Reply:
Thanks for careful review, Petr :) It seems I'm a bit distracted.
The reason for this oversight is that I've been doing stuff like this
```fsharp
[<Theory>]
[<Repeat(5000)>]
let ``We can cancel a job`` _ =
```
To repro failures locally.